### PR TITLE
Movies detail

### PR DESCRIPTION
**Pull request by m4dzix**

### CODE REVIEW

File: src/features/Movies/moviesSlice.js [hunk 1]
```diff
@@ -38,13 +56,21 @@ export const {
...
 export const selectMoviesState = (state) => state.movies;
 export const selectPopularMovies = (state) => selectMoviesState(state).movies;
 
 export const selectGenres = (state) => selectMoviesState(state).genres;
-
+export const selectMovieDetails = (state) => selectMoviesState(state).movie;
+export const selectChangeTileStyle = (state) =>
+  selectMoviesState(state).changeTileStyle;
+export const selectLoadingMovieDetails = (state) =>
+  selectMoviesState(state).loading;
```
Comment (KacperSzemraj) [inline src/features/Movies/moviesSlice.js:72]: Surely Prettier formatted the selectors this way? ;)

File: src/features/Movies/moviesSlice.js [hunk 2]
```diff
@@ -38,13 +56,21 @@ export const {
...
   fetchGenresSuccess,
   fetchGenresError,
+  fetchMovieDetails,
+  fetchMovieDetailsSuccess,
+  fetchMovieDetailsError,
+  showId,
 } = moviesSlice.actions;
 
 export const selectMoviesState = (state) => state.movies;
 export const selectPopularMovies = (state) => selectMoviesState(state).movies;
```
Comment (KacperSzemraj) [inline src/features/Movies/moviesSlice.js]: Empty line ;)

File: src/features/People/PersonDetails/index.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+} from "../../Movies/moviesSlice";
+
+const PersonDetails = () => {
+  const personDetails = useSelector(selectPersonDetails);
+  const movieCast = useSelector(selectMovieCast);
+  const movieCrew = useSelector(selectMovieCrew);
+  const movieGenres = useSelector(selectGenres);
+  const loading = useSelector(selectLoading);
+  const castNumber = movieCast.length;
+  const crewNumber = movieCrew.length;
```
Comment (KacperSzemraj) [inline src/features/People/PersonDetails/index.js:33]: Cool, neat ;)

File: src/features/People/PopularPeople/index.js [hunk 4]
```diff
@@ -34,7 +34,11 @@ const PopularPeople = () => {
               changeTileStyle={changeTileStyle}
               key={people.id}
               title={people.name}
-              imagePath={`https://image.tmdb.org/t/p/w185/${people.profile_path}`}
+              imagePath={
+                !!people.profile_path
+                  ? `https://image.tmdb.org/t/p/w185/${people.profile_path}`
+                  : profile
+              }
```
Comment (KacperSzemraj) [inline src/features/People/PopularPeople/index.js:41]: Good job!

File: src/features/People/peopleSlice.js [hunk 5]
```diff
@@ -19,17 +22,40 @@ const peopleSlice = createSlice({
...
   fetchPopularPeopleSuccess,
   fetchPopularPeopleError,
+  fetchCredits,
+  fetchCreditsError,
+  fetchCreditsSuccess,
 } = peopleSlice.actions;
 export const selectPeopleState = (state) => state.people;
 export const selectPopularPeople = (state) => selectPeopleState(state).people;
 export const selectChangeTileStyle = (state) =>
   selectPeopleState(state).changeTileStyle;
```
Comment (KacperSzemraj) [inline src/features/People/peopleSlice.js]: Prettier?

File: src/features/People/peopleSlice.js [hunk 6]
```diff
@@ -19,17 +22,40 @@ const peopleSlice = createSlice({
...
+export const selectCredits = (state) => selectPeopleState(state).credits;
+
```
Comment (KacperSzemraj) [inline src/features/People/peopleSlice.js]: Empty line ;)

File: src/features/People/peopleSlice.js [hunk 7]
```diff
@@ -19,17 +22,40 @@ const peopleSlice = createSlice({
...
+export const selectCrew = (state) => selectPeopleState(state).crew;
+export const selectLoadingCredits = (state) => selectPeopleState(state).loading;
+export const selectCast = (state) => selectPeopleState(state).cast;
+
```
Comment (KacperSzemraj) [inline src/features/People/peopleSlice.js]: Empty line

**Review by KacperSzemraj — APPROVED** [review]
Hi!

I looked through all the changes, and noticed empty lines in some places. Generally the code is understandable, so you can merge.

Kind regards! Kacper

File: src/features/People/peopleSlice.js (hunk 5 above)
Comment (m4dzix) [reply src/features/People/peopleSlice.js]: 😄

File: src/features/Movies/moviesSlice.js (hunk 1 above)
Comment (m4dzix) [reply src/features/Movies/moviesSlice.js:72]: Yes 😉

**Pull request merged**